gwsumm/plot/core.py

```diff
@@ -324,6 +324,8 @@ def pid(self):
             filts = "".join(map(str,
                 [getattr(c, 'filter', getattr(c, 'frequency_response', ''))
                  for c in self.channels]))
+            if self.filterstr:
+                filts = "".join([filts,self.filterstr])
```

Comment:
Just so we're on the same page, you're suggesting that instead of inheriting the pid from the parent class here: `https://github.com/gwpy/gwsumm/blob/master/gwsumm/plot/triggers.py#L122`, I should have it define and use the method above?